src/TodoListItem.jsx

```diff
@@ -0,0 +1,5 @@
+function TodoListItem(props) {
+  return <li key={props.id}>{props.todo}</li>;
```

Comment:
As mentioned in the previous comment, `todo` is an object. It contains `title`, so you should use it like that:

```suggestion
  return <li key={props.id}>{props.todo.title}</li>;
```

You only need to define `key` when you are rendering an array of elements, which it isn't the case here. You can remove it.

```suggestion
  return <li>{props.todo.title}</li>;
```